apps/projects/templates/a4_candy_projects/project_detail.html

```diff
@@ -107,7 +106,8 @@ <h5>{% trans 'Contact for questions' %}</h5>
         <div class="container">
             <div class="row">
                 <div class="col-12">
-                    <h2 class="u-serif-header text-center">{% trans "Statistics" %}</h2>
+                    <h3 class="u-serif-header text-center">{% trans "Statistics" %}</h3>
+                    <p class="text-center text-muted">{% trans "The answered questions are included in the statistics. It shows which groups are represented." %}"</p>
                     <div>
```

Comment:
I think the " behind the curly closing bracket is too much

Reply:
I would agree!